src/issue-filing/common/create-file-issue-handler.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { BrowserAdapter } from '../../background/browser-adapters/browser-adapter';
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { IssueFilingServicePropertiesMap } from '../../common/types/store-data/user-configuration-store';
+import { IssueFilingUrlProvider } from '../types/issue-filing-service';
+
+export const createFileIssueHandler = <Settings>(
+    urlProvider: IssueFilingUrlProvider<Settings>,
+    settingsGetter: (data: IssueFilingServicePropertiesMap) => Settings,
```

Comment:
should this be getSettings? method names should be verb. this is not a class

Reply:
Make sense